src/utils/general.utils.ts

```diff
@@ -368,7 +368,7 @@ export const isTestnetChain = (chainId: string) => {
     return isTestnet
 }
 
-export const areTokenAddressesEqual = (address1: string, address2: string): boolean => {
+export const areAddressesEqual = (address1: string, address2: string): boolean => {
```

Comment:
thought: "address" could be confusing (street address, bank address etc). Not sure what name would be better though

Reply:
renaming to areEvmAddressesEqual